DESCRIPTION

```diff
@@ -33,7 +33,7 @@ Depends: R (>= 3.6.0)
 Imports:
     stats
 Suggests:
-    testthat (>= 3.0.0),
+    testthat (>= 3.2.0),
```

Comment:
@justinmcgrath I only just noticed this change, tagged on after all the other changes, because it required me to upgrade.  Was there a reason for it?